app/javascript/src/NewService/components/NewServiceForm.jsx

```diff
@@ -24,8 +27,8 @@ const NewServiceForm = (props: Props) => {
     setFormMode(event.currentTarget.value)
   }
 
-  const formToRender = () => formMode === 'manual' || !isServiceDiscoveryAccessible
```

Comment:
I remember it (`|| !isServiceDiscoveryAccessible`) was needed to not show Service Discovery feature in SAAS.
@josemigallas you made this fix, remember?
Or it's not needed anymore?

Reply:
I remember that was added after.. but analyzing the current logic, I think it might be redundant. `formMode` is `'manual'` by default and this **cannot change** when `!isServiceDiscoveryAccessible`. This happens because the radio buttons only render when it is `true`.

In other words: `formMode === 'service'` and `isServiceDiscoveryAccessible === false` is not possible, I think.

Reply:
OK